vzstorage-pd/vendor/github.com/avagin/ploop-flexvol/volume/volume.go

```diff
@@ -1,95 +0,0 @@
-package volume
-
-import (
-	"fmt"
-	"os"
-	"os/exec"
-
-	"github.com/dustin/go-humanize"
-	"github.com/kolyshkin/goploop-cli"
-)
-
-func Create(options map[string]string) error {
```

Comment:
you should create a separate patch to move code from this file. Separate each logical change into a separate patch.